apps/test/unit/templates/rubrics/EvidenceLevelsTest.jsx

```diff
@@ -32,6 +42,13 @@ describe('EvidenceLevels', () => {
     expect(wrapper.find('Heading6').props().children).to.equal('Rubric Scores');
     expect(wrapper.find('Memo(RadioButton)').length).to.equal(0);
     // Two BodyThreeText per evidence level
-    expect(wrapper.find('BodyThreeText').length).to.equal(4);
+    expect(wrapper.find('BodyThreeText').length).to.equal(
+      DEFAULT_PROPS.evidenceLevels.length * 2
```

Comment:
Again, just trying to make this less fragile.  No impact to what is actually being tested here.